server/core/connector.go

```diff
@@ -302,6 +312,64 @@ func (conn *BridgeConnector) subscribeToChannel() (stan.Subscription, error) {
 	return sub, err
 }
 
+// set up a JetStream subscription, assumes the lock is held
+func (conn *BridgeConnector) subscribeToJetStream(subject string) (*nats.Subscription, error) {
+	if conn.bridge.JetStream() == nil {
+		return nil, fmt.Errorf("bridge not configured to use JetStream")
+	}
+
+	options := []nats.SubOpt{}
+
+	if conn.config.DurableName != "" {
+		options = append(options, nats.Durable(conn.config.DurableName))
+	}
+
+	if conn.config.StartAtTime != 0 {
+		t := time.Unix(conn.config.StartAtTime, 0)
+		options = append(options, nats.StartTime(t))
+	} else if conn.config.StartAtSequence == -1 {
+		options = append(options, nats.DeliverLast())
+	} else if conn.config.StartAtSequence > 0 {
+		options = append(options, nats.StartSequence(uint64(conn.config.StartAtSequence)))
+	} else {
+		options = append(options, nats.DeliverAll())
+	}
+
+	options = append(options, nats.AckExplicit())
```

Comment:
I think flow control might be useful in this case, would help with use cases like reading from a kafka topic with lots of messages

Reply:
Added flow control. 👍🏽